kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/OperationSupport.java

```diff
@@ -579,7 +585,8 @@ private <T> T handleResponse(HttpRequest.Builder requestBuilder, Class<T> type,
    *
    * @return Returns a de-serialized object as api server response of provided type.
    */
-  protected <T> CompletableFuture<T> handleResponse(HttpClient client, HttpRequest.Builder requestBuilder, Class<T> type,
+  protected <T> CompletableFuture<T> handleResponse(HttpClient client, HttpRequest.Builder requestBuilder,
```

Comment:
Wouldn't it be easier to provide an overloaded method instead of duplicating the TypeReference inner-class instantiation in most of the calling methods?

Reply:
I don't like how many overloaded methods there are already in OperationSupport, so it didn't seem worth it as it would just combine 2 calls.

Reply:
sounds good